src/controller.py

```diff
@@ -121,3 +151,12 @@ def end_turn(self):
             'Infect phase gone. Starting new turn.')
 
         self._switch_player()
+
+    def _switch_player(self):
+        self.current_player = self.players[next(self.name_cycle)]
+        logging.info(
+            f'Active player: {self.current_player.name}')
+
+        self.game.start_turn(self.current_player)
```

Comment:
For future refactoring: `start_turn` is a Game method, while `end_turn` is in the Controller. We might need to rename it or remove it altogether. It virtually does nothing: resets action number to 4.

Reply:
Added this to #13.
Both methods are worth being left as they could handle events happening on the player's turn.